operators/tokenizer/bpe_json.hpp

```diff
@@ -9,6 +9,8 @@
 
 #include "bpe_types.h"
 
+#include <filesystem>
```

Comment:
it looks like we don't use any fancy feature from filesystem, can we remove the dependency now and use file_sys.h instead?

Reply:
yep! good catch, removed.